apps/wallet-mobile/src/yoroi-wallets/cardano/constants/common.ts

```diff
@@ -58,4 +58,5 @@ export const HISTORY_REFRESH_TIME = 35000
 
 export const COINS_PER_UTXO_WORD = '34482'
 
+// TODO: This should come from protocolParams endpoint
```

Comment:
@pedromtcosta any estimate for BE zero to be updated?

Reply:
We do have the endpoint, not sure if there's another update is necessary on BE. Maybe for the caching so ogmios doesn't struggle so much.
I changed useLockedAmount to use the data from the endpoint, but I think there's other places in the app that still use this (and maybe other?) constant and should all be changed.